packages/common/src/index.ts

```diff
@@ -468,7 +488,7 @@ export default class Common extends EventEmitter {
       if (hf['name'] === hardfork2) posHf2 = index
       index += 1
     }
-    return posHf1 >= posHf2
+    return posHf1 >= posHf2 && posHf2 !== -1
```

Comment:
Ok, I thought about this a bit longer on eventual side effects and also in-between hardforks which didn't activate on some chains (e.g. `muirGlacier` on the PoA chains) and after some back-and-forth I cautiously think I can go along with this change, still a bit unsure though. An in between thought I had was if we should rather throw if one of the two HFs provided are not present. This will likely make things more unstable though I guess and might cause unintentional throws where the code would have otherwise proceeded without errors.

Nevertheless writing this down so explicitly here so that others might give this some thought as well if needed.

Reply:
Yeah I agree this is a very big change. However, if we start enforcing these things that means that every time we implement a new EIP and people use their custom chains then if they update their packages it suddenly does not work anymore, since they have to explicitly state that they do not support the EIP. I would say that (before this change) it was an opt-out behavior, but it should actually be opt-in: we assume that if EIPs are not present in some common, they are not active.